samples/interactive-tutorials/src/main/java/product/ImportProductsGcs.java

```diff
@@ -48,7 +48,7 @@ public static void main(String[] args) throws IOException, InterruptedException
     String gcsErrorBucket = String.format("%s/errors", gcsBucket);
     String gscProductsObject = "products.json";
     // TO CHECK ERROR HANDLING USE THE JSON WITH INVALID PRODUCT
```

Comment:
nit: I know this wasn't part of your original PR but since you're editing it, can you update this comment to not be all caps and fix the grammar? Something like `To check error handling, use an invalid product JSON.`

Reply:
Refactored accordingly.